docs/install.rst

```diff
@@ -25,32 +25,29 @@ Start by getting the source::
 
 .. Important::
 
-    Because Bedrock uses submodules, it is important not only to use
+    Because Bedrock uses submodules for legal-docs, it is important not only to use
     ``--recursive`` on the initial clone, but every time you checkout
-    a different branch, to update the submodules with::
+    a different branch if you need to work with legal-docs, to update the submodules with::
 
         git submodule update --init --recursive
```

Comment:
While I have an alias to do this I hardly ever needed it even when we did have more than just the one git submodule, and having this advice in the docs didn't actually prevent issues. It might be better to simplify the docs for the main target audience-- those installing bedrock for the first time-- and just add "to get the legal-docs submodule" to the end of the "make sure you use --recursive" parenthetical above and eliminate this "Important" section entirely.